staking/programs/staking/src/state/vesting.rs

```diff
@@ -25,6 +25,11 @@ pub enum VestingSchedule {
         period_duration: u64,
         num_periods:     u64,
     },
+    PeriodicVestingAfterListing {
```

Comment:
I think something awkward about the current implementation is that the first unlock is at `start_date + 1 epoch`. We can fix that in a subsequent PR.

Reply:
hm good point. let me double check what the exact schedule is.